config.h

```diff
@@ -213,6 +213,19 @@
 
 #endif
 
+// If defined, enable a low batt icon triggered by a GPIO pin whose BCM number is given.
+// #define LOW_BATT_PIN 19
+
+#if defined(LOW_BATT_PIN)
+
+// Which state of the LOW_BATT_PIN is considered to be low batt (1 is up, 0 is down).
+#define LOW_BATT_STATE 0

```

Comment:
Would you be open to changing the naming across this PR from `LOW_BATT_*` to `LOW_BATTERY_*`? It is not that much longer, but reads a bit more easier?